lib/rufo/command.rb

```diff
@@ -82,7 +82,7 @@ def format_file(filename)
     code = File.read(filename)
 
     begin
-      result = format(code, File.dirname(filename))
+      result = format(code, @filename_for_dot_rufo || File.dirname(filename))
```

Comment:
Totally forgot: @danielma brought us [some nice specs for command.rb](https://github.com/ruby-formatter/rufo/blob/fe4eedf2400dca774e231083fdf9324514d00ea7/spec/command_spec.rb).
Is there is a way to add spec for the change above? :neckbeard: